addons/hr_expense/models/hr_expense.py

```diff
@@ -102,6 +103,14 @@ def _compute_state(self):
             else:
                 expense.state = "done"
 
+    @api.depends('sheet_id', 'sheet_id.accounting_date')
+    def _compute_accounting_date(self):
```

Comment:
Isn't it a related then ?

Reply:
Indeed! Not sure why I left it like that :man_facepalming: